app/posts/manage-teacher-training-applications/2021-03-18-self-service-registration.md

```diff
@@ -0,0 +1,436 @@
+
+We need to:
+
+- reduce the amount of manual work done by the transition team, to save team effort and reduce the risk of errors
+- know which HEIs are on Manage
+- know which SDs have been set up for each HEI
+- know which SDs have set up and opened courses on Manage
+
+## Provider data analysis
+
+There are 87 HEIs in the Manage database. 22 HEIs have been set up to use Manage and the remaining 65 have not.
+
+### Number of SDs connected to each HEI
+
+Using the Publish database, we looked at how many SDs are connected to each HEI.
+
+- 25% of HEIs have no SDs
+- 24% of HEIs have between 1 and 10 SDs
+- 36% of HEIs have between 11 and 20 SDs
+- 15% of HEIs have more than 20 SDs
+- Average number of SDs per HEI: 10
+- Maximum number of SDs per HEI: 33
+- Minimum number of SDs per HEI: 0
+
+If we ask an HEI user separately about each SD then on average they will see 10 pages of questions.
+
+### Number of HEIs connected to each SD
+
+- 77% of SDs have only 1 HEI
+- 14% have 2 HEIs
+- 6% have 3 HEIs
+- 3% have more than 3 HEIs
+
+This is important since if an SD has multiple HEIs, they only need to be onboarded once. The other HEIs accrediting their courses will not need to provide contact details for them.
+
+### Cross-over between users of Manage and Publish
+
+#### Manage users in publish
+
+- 59% of Manage users are in Publish
+- 68% of active Manage users are in Publish
+- 63% of active Manage users are active in Publish
+
+#### Publish users in Manage
+
+- 20% of Publish users are also in Manage
+- 27% of active Publish users are in Manage
+- 23% of active Publish users are active in Manage
+
+Since there are a lot of users in Publish who are not set up in Manage, we have a large pool of contact details to draw from. These users will already have DfE Sign In accounts.
+
+## Iteration 1
+
+We decided to start by prototyping a small service which would allow HEI users to provide us with the information we need.
+
+### What we did and why
+
+In our first iteration, we wanted HEI users to answer two questions.
+
+- Do you want to open courses at the training provider?
+- Who do you want to manage teacher training applications at the provider?
+
+This would allow us to invite the right people at each SD, while leaving the HEI in control of when their courses become available.
+
+{{ appFigure({
+  image: {
+    file: "self-service-registration-v1-flow.png",
+    alt: "Iteration 1 user flow"
+  },
+  caption: "Figure 1: Iteration 1 user flow"
+}) }}
+
+The journey began on a ‘start page’ listing the SDs working with the HEI and setting out the steps involved in creating an account.
+
+The aim was to give the user a chance to gather information before starting, which is important since it is not possible to save and return to the process.
+
+Users would reach the start page from an email. For this iteration we did not include DfE Sign In account creation, as we assumed that the email would contain a ‘magic link’ bypassing it.
```

Comment:
Change DfE Sign In to DfE Sign-in everywhere

Reply:
I have updated as suggested.